app/shared/types/commonSchemas.ts

```diff
@@ -20,6 +20,7 @@ export const attachmentSchema = {
     originalname: { type: 'string' },
     filename: { type: 'string' },
     mimetype: { type: 'string' },
+    url: { type: 'string' },
```

Comment:
Given attachmentSchema is already not being used anywhere, is it safe to remove this schema?